mne/beamformer/_compute_beamformer.py

```diff
@@ -154,6 +154,27 @@ def _normalized_weights(Wk, Gk, Cm_inv_sq, reduce_rank, nn, sk):
     norm *= sk[:, np.newaxis, :]
     power = np.matmul(norm, np.matmul(Wk, Gk))  # np.dot for each source
 
+   # Determine orientation of max power
+    Wk = _pick_ori(power, Wk, Gk, Cm_inv_sq, nn)
+
+
+def _pick_ori(power, Wk, Gk, Cm_inv, nn):
+    """Pick the orientation that maximizes output power.
+
+    Parameters
+    ----------
+    power : ndarray, shape (n_sources, 3, 3)
+        Power source estimate.
```

Comment:
The dimensions of power, are they expected to be n_sources x 3 x 3 - anyone?

Reply:
Yes. Answered in code in the form of a 3-line commit push with `assert` statements :)

Reply:
Okay, I am drawing a mental blank ... Shouldn't it be one value per source per orientation?

Reply:
It is actually the source covariance/CSD at that point. So power on the diagonals and co-variance on the off-diagonals. Orientation of max-power from the covariance/CSD should be the largest eigenvector.

Reply:
Aha, we should rename at least the descriptions then.